src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -115,6 +115,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
+    String purpose = request.getHeader("purpose");
```

Comment:
This is fine for now, but generally when your GET handler is serving too many different purposes, it's probably better to break it down to multiple handlers.
Ideally, we'd have a GET handler for each resource. So a handler at `/conversations/<convo_id>/members` would return the list of members. We can then also have a POST handler at the same endpoint to set those members.

No action needed for here though, just discussing a more widely accepted convention. :)

Reply:
yeah that would have made things much more organized.